src/parse.js

```diff
@@ -43,13 +44,16 @@ const getComparisonCommitHashes = ({
  *
  * @param message
  * @param {object} options
- * @param {Function} options.getCommitType
  * @param {boolean} options.isAllowNonConventionalCommits
- * @returns {{description: string, type: string, prNumber: string, hash: *}|{scope: string, description: string, type: string, prNumber: string, hash: string, typeScope: string}}
+ * @param {object} settings
+ * @param {Function} settings.getCommitType
+ * @returns {{description: string, type: string, prNumber: string, hash: *}|{scope: string, description: string,
+ *     type: string, prNumber: string, hash: string, typeScope: string}}
  */
 const parseCommitMessage = (
   message,
-  { getCommitType: getAliasCommitType = getCommitType, isAllowNonConventionalCommits } = {}
+  { isAllowNonConventionalCommits } = OPTIONS,
+  { getCommitType: getAliasCommitType = getCommitType } = {}
 ) => {
   const commitType = getAliasCommitType({ isAllowNonConventionalCommits });
```

Comment:
required: confirm these funcs aren't passing options, may be able to cut down on additional drilling